e2e/affix.e2e-spec.ts

```diff
@@ -85,12 +85,12 @@ describe('Affix', function () {
     await takeScreenshots('window');
     await element(by.id('screenshot-affix-button-overflow-parent')).click();
     done();
-  });
+  }, 60000);
```

Comment:
Visual tests consistently timeout because each `it` is taking several screenshots; increasing the timeout should alleviate that.